### ISSUE TITLE
Attention is test !  Bug : boss is missing ...

### ISSUE THREAD

**Issue by Zydrax** [Dreadox/HighRisk-BugTracker#3]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote )

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

boss is missing
and why i cant tmog warr items ?

**Expected behaviour**: Tell us what should happen instead

tmog warrior items 

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

[https://imgur.com/a/WkeNnfX](url)

**Comment by Matilly0512** [comment]
Possible Issue: Gear(green) Doesn't show the stats it has. unless i shift, Mouse over it.


![WoWScrnShot_121921_112745](https://user-images.githubusercontent.com/96388080/146684628-7f16d623-d0f3-43a8-a049-3c0a6b8d5bb8.jpg)
![WoWScrnShot_121921_112740](https://user-images.githubusercontent.com/96388080/146684632-90cd0dbd-0656-4d4b-a616-3d94dcb83583.jpg)

**Comment by Matilly0512** [comment]
Quest NPCs(Elementals) to powerful. 
Quest is a Mage quest in Stormwind from the Mage Tower. 
Name: Investigate the Blue Recluse

Could be gear issue
But it's a gray quest at lvl 24.
